lib/ios/RNNBottomTabsController.m

```diff
@@ -54,16 +56,18 @@ - (instancetype)initWithLayoutInfo:(RNNLayoutInfo *)layoutInfo
     return self;
 }
 
+- (void)createTabBarItems:(NSArray<UIViewController *> *)childViewControllers {
+    for (UIViewController *child in childViewControllers) {
```

Comment:
you might need to limit tabs for 5, be aligned with Android.

In android we cant have more than 5 tabs, RN apps will use the same amount of tabs for ios and for android, so apps with more than 5 tabs will crash the Android clients.

Reply:
That's what the `More` tab is for. We have this natively supported on iOS.